Gemfile

```diff
@@ -7,6 +7,7 @@ gem 'activesupport', '~> 4.2'
 gem 'railties', '~> 4.2'
 gem 'rails-api', '0.4.0'
 gem 'jsonite', '0.0.3'
+gem 'geokit-rails'
```

Comment:
This looks useful. We might be able to use IP location lookup if a client doesn't provide lat/lon.